runtime/parachains/src/runtime_api_impl/v1.rs

```diff
@@ -231,6 +232,21 @@ pub fn session_index_for_child<T: initializer::Config>() -> SessionIndex {
 	<shared::Module<T>>::session_index()
 }
 
+pub fn get_historical_authority_ids<T: initializer::Config>() -> Vec<AuthorityDiscoveryId> {
```

Comment:
```suggestion
pub fn get_historical_authority_ids<T: session_info::Config>() -> Vec<AuthorityDiscoveryId> {
```

Reply:
@ordian Can't do that. Since, session_index_for_child requires `initializer` module.

Reply:
right, this was assuming we remove the `session_index_for_child` call completely, but it's fine if we don't